sources/web/datalab/templates/nb.html

```diff
@@ -161,6 +161,15 @@
                 <li id="interruptKernelButton"><a href="#">Interrupt Execution</a></li>
               </ul>
             </div>
+            <div class="btn-group">
+              <button id="kernelSelectorButton" type="button" data-toggle="dropdown" class="toolbar-btn" title="Select kernel">
+                <i class="material-icons">track_changes</i>
+                <span class="toolbar-text">Kernel: </span>
+                <span id="currentKernelName"></span>
+              </button>
+              <ul class="dropdown-menu" id="kernelSelectorDropdown">
```

Comment:
can you add a screenshot for this to start discussion on the design? I didn't actually put much thought into how usable this is at the time I made those changes, it was the simplest thing to do for switching kernels.